packages/compass-e2e-tests/tests/instance-sidebar.test.ts

```diff
@@ -186,4 +186,14 @@ describe('Instance sidebar', function () {
       .$(Selectors.databaseTab('Collections', true))
       .waitForDisplayed();
   });
+
+  it('can refresh the databases', async function () {
+    const db = `db_${Date.now()}`;
+    const coll = `coll_${Date.now()}`;
+    await browser.executeInShell(`use ${db};`);
```

Comment:
I probably wouldn't have bothered adding it via the shell since we usually have a client somewhere in tests where we can just run commands against the DB, but this is nice in that it mimicks the common use case 👍

Reply:
So I started looking for a client to otherwise execute it without my helper and realised that there is already `shellEval` supposedly doing the same stuff as `executeInShell`. I will probably just used that and get rid of my helper.

Reply:
Oh yeah you're right. I totally forgot about that. I think it was one of the first few commands we had.

Reply:
perfect - changes are ready for another review 😄